charts/k8s-monitoring/charts/feature-pod-logs/templates/_common_pod_discovery.alloy.tpl

```diff
@@ -75,6 +75,23 @@ discovery.relabel "filtered_pods" {
     target_label = "service_name"
   }
 
+  // explicitly set service_namespace.
+  //
+  // choose the first value found from the following ordered list:
+  // - pod.annotation[resource.opentelemetry.io/service.namespace]
+  // - pod.namespace
```

Comment:
👍 this is compliant with [Semantic conventions 1.35.0 / Non-normative / K8s attributes
Specify resource attributes using Kubernetes annotations](https://opentelemetry.io/docs/specs/semconv/non-normative/k8s-attributes/).